### PR TITLE
CO-1452 - Compatibility between hr_attendance_calendar and hr_switzerland

### PR DESCRIPTION
**Pull request by samuel-fringeli**

### CODE REVIEW

File: hr_switzerland/models/hr_employee.py [hunk 1]
```diff
@@ -12,72 +12,76 @@
...
         for employee in self:
-            employee.today_hour = employee._convert_hour_to_time(
-                employee._today_hour())
-
-    def _compute_bonus_malus_hour(self):
+            current_att_day = self.env['hr.attendance.day'].search([
+                ('employee_id', '=', employee.id),
+                ('date', '=', fields.Date.today())])
+            employee.today_hour = \
+                current_att_day.worked_hours - current_att_day.due_hours
```
Comment (eicher31) [inline hr_switzerland/models/hr_employee.py]: current_att_day.worked_hours doesn't return the current time elapsed, you need the previously method used for compute the employe.today_hour

File: hr_switzerland/models/hr_employee.py [hunk 2]
```diff
@@ -12,72 +12,76 @@
...
+    @api.multi
+    def _compute_extra_hours(self):
         for employee in self:
-            employee.bonus_malus_hour = '-' if employee.bonus_malus < 0 else ''
-            employee.bonus_malus_hour += \
-                employee._convert_hour_to_time(math.fabs(employee.bonus_malus))
+            employee.extra_hours_formatted = \
+                '-' if employee.extra_hours < 0 else ''
+            employee.extra_hours_formatted += \
+                employee._convert_hour_to_time(math.fabs(employee.extra_hours))
```
Comment (eicher31) [inline hr_switzerland/models/hr_employee.py]: You can replace math.fabs by abs() without import (to verify)

**Review by eicher31 — CHANGES_REQUESTED** [review]

**Comment by ecino** [comment]
@eicher31 are you ok with this pull request?

File: partner_communication_switzerland/migrations/10.0.2.4.1/post-migration.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+#    @author: Samuel Fringeli <samuel.fringeli@me.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from openupgradelib import openupgrade
+import re
+
+# on doit pouvoir faire en sorte que les mails déjà existants à qui on a envoyé
```
Comment (ecino) [inline partner_communication_switzerland/migrations/10.0.2.4.1/post-migration.py]: Please write comments in English, as it is hosted and public on github.

**Review by eicher31 — APPROVED** [review]

**Comment by eicher31** [comment]
pep8 error
partner_communication_switzerland/migrations/10.0.2.4.1/post-migration.py:19:1: E302 expected 2 blank lines, found 1

**Review by ecino — CHANGES_REQUESTED** [review]
Please fix it with PEP8 error. Thanks

**Review by ecino — APPROVED** [review]

**Pull request merged**